git-duet/main.go

```diff
@@ -67,6 +67,10 @@ func main() {
 		printNextComitter(committers)
 		if configuration.CoAuthoredBy {
 			installPrepareCommitMsgHook()
+			if err = gitConfig.SetAuthor(author); err != nil {
```

Comment:
Can you clarify why this is needed? It wasn't immediately apparent to me (maybe a comment here would help).

Reply:
This is needed for the case that the user runs `GIT_DUET_CO_AUTHORED_BY=1 git duet` without args (author initials) and without having set `GIT_DUET_SET_GIT_USER_CONFIG` in previous `git duet` commands. If the above line is missing, [this test](https://github.com/ansd/git-duet/blob/cf30c43de08bfe373d4e84620e5fd14496af5971/test/git-duet.bats#L207-L221) will fail. I will add a comment for this.

The basic question is whether running `git duet` without args (author initials) should install the hook (and set the author like above) or should it be just a "getter" like it always has been which merely prints the current author and committer?